coderd/database/dbauthz/querier.go

```diff
@@ -1360,10 +1333,10 @@ func (q *querier) InsertWorkspace(ctx context.Context, arg database.InsertWorksp
 	return insert(q.log, q.auth, obj, q.db.InsertWorkspace)(ctx, arg)
 }
 
-func (q *querier) InsertWorkspaceBuild(ctx context.Context, arg database.InsertWorkspaceBuildParams) (database.WorkspaceBuild, error) {
+func (q *querier) InsertWorkspaceBuild(ctx context.Context, arg database.InsertWorkspaceBuildParams) (database.WorkspaceBuildThin, error) {
 	w, err := q.db.GetWorkspaceByID(ctx, arg.WorkspaceID)
 	if err != nil {
-		return database.WorkspaceBuild{}, err
+		return database.WorkspaceBuildThin{}, err
 	}
```

Comment:
Since you still have the workspace here, this doesn't need to be thin right? Or is it a requirement because of the sqlc generating the interface?

Reply:
It's because of SQLc interface